pkg/cloudprovider/providers/aws/aws_loadbalancer.go

```diff
@@ -90,7 +90,7 @@ func (s *AWSCloud) ensureLoadBalancer(namespacedName types.NamespacedName, loadB
 				request := &elb.AttachLoadBalancerToSubnetsInput{}
 				request.LoadBalancerName = aws.String(loadBalancerName)
 				request.Subnets = stringSetToPointers(additions)
-				glog.V(2).Info("Attaching load balancer to added subnets")
+				glog.V(2).Infof("Attaching load balancer for %v to added subnets %v", request.LoadBalancerName, request.Subnets)
```

Comment:
I think the subnets output will be useless, because it is a []*string (right?).  If so, can we just remove both logging statement changes.  That should also make this easier to backport (if we just have the actual fix in aws_utils.go)